adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -1282,6 +1454,20 @@ func getCorsPolicy(corsConfig *model.CorsConfig) *routev3.CorsPolicy {
 func genRouteCreateParams(swagger *model.MgwSwagger, resource *model.Resource, vHost, endpointBasePath string,
 	prodClusterName string, sandClusterName string, requestInterceptor map[string]model.InterceptEndpoint,
 	responseInterceptor map[string]model.InterceptEndpoint, organizationID string) *routeCreateParams {
+
+	var rlMethodDescriptorValue string
+	var isRLPolicyAvailable bool = false
+	if swagger.RateLimitLevel == RateLimitPolicyAPILevel {
+		rlMethodDescriptorValue = RateLimitPolicyAPILevel
```

Comment:
shall we set isRLPolicyAvailable to true in this `if` block too?

Reply:
fixed with: https://github.com/wso2/product-microgateway/pull/3142/commits/72436f53cfb0d91ef4ab61fe2dac061a8a4ca4f5